src/pickup.cpp

```diff
@@ -818,18 +837,20 @@ void Pickup::pick_up( const tripoint &pos, int min )
             }
 
             int pw = pickupW;
-            const char *unmark = _("[left] Unmark");
-            const char *scroll = _("[up/dn] Scroll");
-            const char *mark   = _("[right] Mark");
-            mvwprintw(w_pickup, maxitems + 1, 0,                         unmark);
-            mvwprintw(w_pickup, maxitems + 1, (pw - std::strlen(scroll)) / 2, scroll);
-            mvwprintw(w_pickup, maxitems + 1,  pw - std::strlen(mark),        mark);
-            const char *prev = _("[<] Prev");
-            const char *all = _("[,] All");
-            const char *next   = _("[>] Next");
-            mvwprintw(w_pickup, maxitems + 2, 0, prev);
-            mvwprintw(w_pickup, maxitems + 2, (pw - std::strlen(all)) / 2, all);
-            mvwprintw(w_pickup, maxitems + 2, pw - std::strlen(next), next);
+            // 256 characters is plenty considering the limited space
+            char help_text[256];
+            std::snprintf(help_text, 256, _("[%s] Unmark"), ctxt.get_desc("LEFT", 1).c_str());
+            mvwprintw(w_pickup, maxitems + 1, 0,                                 help_text);
+            std::snprintf(help_text, 256, _("[%s] Help"), ctxt.get_desc("HELP_KEYBINDINGS", 1).c_str());
+            mvwprintw(w_pickup, maxitems + 1, (pw - std::strlen(help_text)) / 2, help_text);
+            std::snprintf(help_text, 256, _("[%s] Mark"), ctxt.get_desc("RIGHT", 1).c_str());
+            mvwprintw(w_pickup, maxitems + 1, pw - std::strlen(help_text),       help_text);
+            std::snprintf(help_text, 256, _("[%s] Prev"), ctxt.get_desc("PREV_TAB", 1).c_str());
+            mvwprintw(w_pickup, maxitems + 2, 0,                                 help_text);
+            std::snprintf(help_text, 256, _("[%s] All"), ctxt.get_desc("SELECT_ALL", 1).c_str());
+            mvwprintw(w_pickup, maxitems + 2, (pw - std::strlen(help_text)) / 2, help_text);
+            std::snprintf(help_text, 256, _("[%s] Next"), ctxt.get_desc("NEXT_TAB", 1).c_str());
+            mvwprintw(w_pickup, maxitems + 2, pw - std::strlen(help_text),       help_text);
```

Comment:
If it's already used at many other places, it should be safe. At least it's more likely safe than `snprintf`, which is currently not used at all.

Even if there is a problem with it (I'm not aware of any), it can be fixed at a central place.